src/app/cookiePreference.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+import {CookiePreferenceService} from "./cookiePreference.service";
+import {CookieService} from "ngx-cookie-service";
+
+describe('CookiePreferenceService', () => {
+
+  const mockCookieService: CookieService = {
+    get: undefined,
+    set: undefined,
+  } as unknown as CookieService;
```

Comment:
if someone knows a simpler way to do mocking using Jasmine or something I'm happy to change it to that - I couldn't quite see how to to do it and this seems to work.